deps/rabbit/src/rabbit_feature_flags.erl

```diff
@@ -219,6 +222,24 @@
 %% A soft required feature flag will be automatically enabled when a RabbitMQ
 %% node is upgraded to a version where it is required.
 
+-type experiment_level() :: unsupported | supported.
+%% The level of support of an experimental feature flag.
+%%
+%% At first, an experimental feature flag is offered to give a chance to users
+%% to try it and give feedback as part of the design and development of the
+%% feature. At this stage, it is unsupported: it must not be enabled in a
+%% production environment and upgrade to a later version of RabbitMQ while
+%% this experimental feature flag is enabled is not supported.
+%%
+%% Then, the experimental feature flag becomes supported. At this point, it is
+%% stable enough that upgrading is guarantied and help will be provided.
```

Comment:
"guarantied" is a word that does exist but you likely mean "guaranteed" here :)

Reply:
Thank you! I never know how to write it…